Matlab/color_Mett.m

```diff
@@ -0,0 +1,185 @@
+    c.Monet = {'#4e6d58'; '#749e89'; '#abccbe'; '#e3cacf'; '#c399a2'; '#9f6e71'; '#41507b'; '#7d87b2'; '#c2cae3'}; 
+    c.Moreau = {'#421600'; '#792504'; '#bc7524'; '#8dadca'; '#527baa'; '#104839'; '#082844'}; 
+    c.Morgenstern = {'#7c668c'; '#b08ba5'; '#dfbbc8'; '#ffc680'; '#ffb178'; '#db8872'; '#a56457'}; 
+    c.Nattier = {'#52271c'; '#944839'; '#c08e39'; '#7f793c'; '#565c33'; '#184948'; '#022a2a'};
+    c.Navajo = {'#660d20'; '#e59a52'; '#edce79'; '#094568'; '#e1c59a'};
+    c.NewKingdom = {'#e1846c'; '#9eb4e0'; '#e6bb9e'; '#9c6849'; '#735852'}; 
+    c.Nizami = {'#dd7867'; '#b83326'; '#c8570d'; '#edb144'; '#8cc8bc'; '#7da7ea'; '#5773c0'; '#1d4497'};
+    c.OKeefe1 = {'#6b200c'; '#973d21'; '#da6c42'; '#ee956a'; '#fbc2a9'; '#f6f2ee'; '#bad6f9'; '#7db0ea'; '#447fdd'; '#225bb2'; '#133e7e'};
+    c.OKeffe2 = {'#fbe3c2'; '#f2c88f'; '#ecb27d'; '#e69c6b'; '#d37750'; '#b9563f'; '#92351e'};
+    c.Peru1 = {'#b5361c'; '#e35e28'; '#1c9d7c'; '#31c7ba'; '#369cc9'; '#3a507f'};
+    c.Peru2 = {'#65150b'; '#961f1f'; '#c0431f'; '#b36c06'; '#f19425'; '#c59349'; '#533d14'};
+    c.Pillement = {'#a9845b'; '#697852'; '#738e8e'; '#44636f'; '#2b4655'; '#0f252f'}; 
+    c.Pissaro = {'#134130'; '#4c825d'; '#8cae9e'; '#8dc7dc'; '#508ca7'; '#1a5270'; '#0e2a4d'}; 
+    c.Redon = {'#5b859e'; '#1e395f'; '#75884b'; '#1e5a46'; '#df8d71'; '#af4f2f'; '#d48f90'; '#732f30'; '#ab84a5'; '#59385c'; '#d8b847'; '#b38711'}; 
+    c.Renoir = {'#17154f'; '#2f357c'; '#6c5d9e'; '#9d9cd5'; '#b0799a'; '#f6b3b0'; '#e48171'; '#bf3729'; '#e69b00'; '#f5bb50'; '#ada43b'; '#355828'}; 
+    c.Robert = {'#11341a'; '#375624'; '#6ca4a0'; '#487a7c'; '#18505f'; '#062e3d'}; 
+    c.Signac = {'#fbe183'; '#f4c40f'; '#fe9b00'; '#d8443c'; '#9b3441'; '#de597c'; '#e87b89'; '#e6a2a6'; '#aa7aa1'; '#9f5691'; '#633372'; '#1f6e9c'; '#2b9b81'; '#92c051'};
+    c.Stevens = {'#042e4e'; '#307d7f'; '#598c4c'; '#ba5c3f'; '#a13213'; '#470c00'}; 
+    c.Tara = {'#eab1c6'; '#d35e17'; '#e18a1f'; '#e9b109'; '#829d44'}; 
+    c.Thomas = {'#b24422'; '#c44d76'; '#4457a5'; '#13315f'; '#b1a1cc'; '#59386c'; '#447861'; '#7caf5c'};
+    c.Tiepolo = {'#802417'; '#c06636'; '#ce9344'; '#e8b960'; '#646e3b'; '#2b5851'; '#508ea2'; '#17486f'}; 
+    c.Troy = {'#421401'; '#6c1d0e'; '#8b3a2b'; '#c27668'; '#7ba0b4'; '#44728c'; '#235070'; '#0a2d46'}; 
+    c.Tsimshian ={'#582310'; '#aa361d'; '#82c45f'; '#318f49'; '#0cb4bb'; '#2673a3'; '#473d7d'};
+    c.VanGogh1 = {'#2c2d54'; '#434475'; '#6b6ca3'; '#969bc7'; '#87bcbd'; '#89ab7c'; '#6f9954'}; 
+    c.VanGogh2 = {'#bd3106'; '#d9700e'; '#e9a00e'; '#eebe04'; '#5b7314'; '#c3d6ce'; '#89a6bb'; '#454b87'}; 
+    c.VanGogh3 = {'#e7e5cc'; '#c2d6a4'; '#9cc184'; '#669d62'; '#447243'; '#1f5b25'; '#1e3d14'; '#192813'};
+    c.Veronese = {'#67322e'; '#99610a'; '#c38f16'; '#6e948c'; '#2c6b67'; '#175449'; '#122c43'};
+    c.Wissing= {'#4b1d0d'; '#7c291e'; '#ba7233'; '#3a4421'; '#2d5380'};
+    
+    % Calculate how many steps in between are needed
+    if size(c.([map]),1) < minColor
```

Comment:
Matlab does not like `c` without an initial definition as struct in before.

Reply:
now it is initialised before